### DISCUSSION TITLE
Deployment of Nx Monorepo apps.

### DISCUSSION THREAD

**Discussion by saidybarry** [vercel/community#579] [Help] [answered]
I have a Nx monorepo with Apollo graphQL server and two react native apps, I want to deploy the Apollo sever on vercel.

**Comment by samcx** [comment, marked as answer]
@saidybarry It is not possible to deploy an Apollo GraphQL Server on Vercel (long running servers in general). Vercel is, however, committed to dynamic frontends coupled with [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) and [Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions).
